source/operations/network-encryption.rst

```diff
@@ -69,16 +72,29 @@ Enabling TLS
         private.key
         public.crt
 
-   You can use the MinIO :minio-git:`certgen <certgen>` to mint self-signed certificates for enabling TLS for evaluating MinIO with TLS enabled.
-   For example, the following command generates a self-signed certificate with a set of IP and DNS SANs associated to the MinIO Server hosts:
+   You can use the MinIO :minio-git:`certgen <certgen>` to mint self-signed certificates for evaluating MinIO with TLS enabled.
+   For example, the following command generates a self-signed certificate with a set of IP and DNS Subject Alternate Names (SANs) associated to the MinIO Server hosts:
 
    .. code-block:: shell
 
       certgen -host "localhost,minio-*.example.net"
 
-   You can place the generated ``public.crt`` and ``private.key`` into the ``/.minio/certs`` directory to enable TLS for the MinIO deployment.
+   Place the generated ``public.crt`` and ``private.key`` into the ``/.minio/certs`` directory to enable TLS for the MinIO deployment.
    Applications can use the ``public.crt`` as a trusted Certificate Authority to allow connections to the MinIO deployment without disabling certificate validation.
 
+   If you are reconfiguring an existing deployment that did not previously have TLS enabled, update ``MINIO_VOLUMES`` to specify ``https`` instead of ``http``.
+   You may also need to update URLs used by applications or clients.
+
+   Self-signed Certificates
+   ~~~~~~~~~~~~~~~~~~~~~~~~
+
+   For self-signed and internally signed certs, place the appropriate CA certificate in a ``{HOME}/.minio/CAs`` directory:
```

Comment:
`appropriate CA certificate` seem vague, would this be a root CA cert? Some other kind of CA cert?

Reply:
self-signed certificates are themselves the CA. But this also goes for any other certificate signed by a non-global authority (e.g. a company internal CA).

I might expand this section to be something like

'Certificates Signed by Independent Certificate Authorities'

and cover self-signed, internal, or private CAs.

Other than that, a CA must sign a CSR to produce the actual certs. So for a given certificate, we need the CA that signed and produced that cert.

```
For self-signed certificates, the certificate private key is typically also the Certificate Authority.
Place that CA in the `.minio/CAs` directory for all nodes.

For certificates signed by internal, private, or other non-global CAs, place the CA in the `.minio/CAs` directory. 
Ensure the CA includes the full chain of trust from the intermediate certificate to root.
```

Something like that.